README.md

```diff
@@ -215,7 +215,8 @@ This project is deployed at github pages you can clone it here. [Please click to
 - Linkedin: [@linkedinhandle](https://www.linkedin.com/in/basir-ahmadi-8aba89263)
 - Email: elham1378basir@gmail.com
   <br>
```

Comment:
- [ ] Your [Live Demo Link](https://elhambasir.github.io/To-Do-List/) is boken, Kindly fix this or remove it 👍🏻

Reply:
I fixed that. Thank you.